octobot_trading/data/order.py

```diff
@@ -18,6 +18,8 @@
 from asyncio import Lock
 
 import math
+
+from octobot_backtesting.api.backtesting import get_backtesting_current_time
```

Comment:
I don't think it's a good idea to import backtesting module here.

Reply:
why?

Reply:
We can wrap it in exchange_simulator.

Reply:
Time managed in exchange and simulated exchange